src/elements/content-explorer/ContentExplorer.js

```diff
@@ -738,6 +734,7 @@ class ContentExplorer extends Component<Props, State> {
         if (selected) {
             selected.selected = false;
         }
+        this.setState({ selected });
```

Comment:
Another good place to avoid state mutation, so instead of mutating the `selected` item in `this.state` and then setting it back, you can simplify this method to something like:
```
const { selected: selectedItem }: State = this.state;
this.setState({ selected: {...selectedItem, selected: false } });
```